modules/apps/document-library/document-library-service/src/test/java/com/liferay/document/library/internal/search/SearchResultUtilDLFileEntryTest.java

```diff
@@ -364,11 +365,16 @@ private SearchResultContributor _createSearchResultContributor() {
 		DLFileEntrySearchResultContributor dlFileEntrySearchResultContributor =
 			new DLFileEntrySearchResultContributor();
 
-		dlFileEntrySearchResultContributor.setClassNameLocalService(
+		ReflectionTestUtil.setFieldValue(
```

Comment:
@JanisZhang Please remove the blank line here, see this comment for more details https://github.com/brianchandotcom/liferay-portal/pull/122468#issuecomment-1233491759